### PR TITLE
clean up readme and docs

### PR DESCRIPTION
**Pull request by jessleenyc**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [ ] Bug Fix
- [x] Documentation Update

## Description
We were starting to have duplicate info between the docs and readme. This shortens the readme but keeps the standard installation, and moves everything else to docs. 

- Remove GitPod installation from readme (it's in docs already)
- Remove Docker installation from readme (its also in docs)
- Point ppl to docs if they want installation guide for GitPod & Docker
- Add engineering standards to docs and remove from readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,25 +64,11 @@ We encourage you to contribute to dev.to! Please check out the [Contributing to
...
-
-For Javascript, we follow [Airbnb's JS Style Guide](https://github.com/airbnb/javascript), using [ESLint](https://eslint.org) and [prettier](https://github.com/prettier/prettier). If you have ESLint installed with your text editor of choice, you should be up and running.
-
-#### Husky hooks
-
-When commits are made, a git precommit hook runs via [husky](https://github.com/typicode/husky) and [lint-staged](https://github.com/okonet/lint-staged). ESLint, prettier, and Rubocop will run on your code before it's committed. If there are linting errors that can't be automatically fixed, the commit will not happen. You will need to fix the issue manually then attempt to commit again.
-
-Note: if you've already installed the [husky](https://github.com/typicode/husky) package at least once (used for precommit npm script), you will need to run `yarn --force` or `npm install --no-cache`. For some reason, the post-install script of husky does not run when the package is pulled from yarn or npm's cache. This is not husky specific, but rather a cached package issue.
+Additional technologies and services are listed on [our docs](https://docs.dev.to/technical-overview/).
```
Comment (Zhao-Andy) [inline README.md]: I think we should have a line like "Detailed information, such as installation guides, tech stack information, etc. live in [our docs](https://docs.dev.to)".

File: README.md [hunk 2]
```diff
@@ -64,25 +64,11 @@ We encourage you to contribute to dev.to! Please check out the [Contributing to
...
 
 ## Getting Started
 
-This section provides a high-level requirement & quick start guide. For detailed installations, please check out our [docs](http://docs.dev.to/installation).
+This section provides a high-level requirement & quick start guide. **For detailed installations, such as getting started with GitPod, Docker, or specific operating systems, please check out our [docs](http://docs.dev.to).**
```
Comment (Zhao-Andy) [inline README.md]: ...and then down here continue to link to /installation here. I know we link to it down below but that's another 30 lines 😅

File: docs/getting-started/readme.md [hunk 3]
```diff
@@ -4,6 +4,8 @@ items:
   - forking.md
   - config-env.md
   - db.md
+  - start-app.md
+  - workflow.md
```
Comment (rhymes) [inline docs/getting-started/readme.md]: Everything seems cool! I can't find start-app.md and workflow.md in the PR though

Comment (jessleenyc) [reply docs/getting-started/readme.md]: GOOD CATCH! just added.

**Pull request merged**